src/components/PartySwitch/PartySwitch.tsx

```diff
@@ -53,17 +53,14 @@ export type PartySwitchProps = {
 };
 
 const CustomDrawer = styled(Drawer)(() => ({
-  "& .MuiDrawer-paper": {
-    width: "30vw",
-  },
-  ["@media only screen and (max-width: 576px)"]: {
+  ["@media only screen and (max-width: 599px)"]: {
     "& .MuiDrawer-paper": {
-      width: "90vw",
+      width: "100vw",
     },
   },
-  ["@media only screen and (min-width: 577px) and (max-width: 992px)"]: {
+  ["@media only screen and (min-width: 600px)"]: {
     "& .MuiDrawer-paper": {
-      width: "40vw",
+      width: "417px",
```

Comment:
I understand that the design project uses this specific value, but what about editing the previous `vw` unit to get a similar result?
```suggestion
      width: "35vw", /* ?? */
```

Reply:
I'm uncertain about this approach. I attempted to make it exactly match the design, but I'm not sure if achieving only a similar appearance is acceptable to the design team.

Reply:
What about setting a `minWidth` and `maxWidth` value: it's fluid but within certain limits. If we set an arbitrary number we should at least round it (`420` instead of `417`?)

Reply:
I rounded up the fixed width to 420.
For screens 1536px and larger, the drawer will now occupy 35vw of the viewport width. This was done so the drawer width is dynamic does not seem to small on very large screens.